src/core/create-ad-slot.ts

```diff
@@ -117,20 +116,24 @@ const concatSizeMappings = (
 	defaultSizeMappings: SizeMapping,
 	optionSizeMappings: SizeMapping = {},
 ): SizeMapping =>
-	Object.entries(optionSizeMappings).reduce<SizeMapping>(
-		(sizeMappings, [breakpoint, optionSizes]) => {
-			// Only perform concatenation if breakpoint is of the correct type
-			if (isBreakpoint(breakpoint)) {
-				const sizes = findAppliedSizesForBreakpoint(
-					sizeMappings,
-					breakpoint,
-				);
-
-				// Concatenate the option sizes onto any existing sizes present for a given breakpoint
-				sizeMappings[breakpoint] = sizes.concat(optionSizes);
+	Object.entries(optionSizeMappings).reduce(
```

Comment:
The previous implementation was not correctly combining the sizeMappings, it would have duplicate sizes and add sizes for smaller breakpoints to larger.

I don't think it actually caused any problems? But the results were unexpected.